charms/worker/k8s/src/literals.py

```diff
@@ -25,7 +25,7 @@
 KUBECTL_PATH = Path("/snap/k8s/current/bin/kubectl")
 K8SD_SNAP_SOCKET = f"{SNAP_COMMON}/var/lib/k8sd/state/control.socket"
 K8SD_PORT = 6400
-SUPPORTED_DATASTORES = ["dqlite", "etcd"]
+SUPPORTED_DATASTORES = ["managed-etcd", "dqlite", "etcd"]
```

Comment:
You can split these constants and make an aggregation of all of them. For example:

```suggestion
DQLITE_DATASTORE= "dqlite"
MANAGED_ETCD_DATASTORE = "managed-etcd"
EXTERNAL_ETCD_DATASTORE = "etcd"
SUPPORTED_DATASTORES = [DQLITE_DATASTORE, MANAGED_ETCD_DATASTORE, EXTERNAL_ETCD_DATASTORE]
```